### DISCUSSION TITLE
[Unstable] Water stream pushes sub away

### DISCUSSION THREAD

**Discussion by FlashORC** [FakeFishGames/Barotrauma#14846] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Trying to get into the water stream from any direction drops the horizontal and(or) vertical velocity to 0, pushes the sub away a bit and does not allow to get into the stream. Noticeable even with slow flow which every sub should overcome.
Observed on 2nd biome.
Vanilla sub is used.
No errors in console.

### Reproduction steps

1. Strart a singleplayer campaign
2. Get to the biome with water streams (e.g. 2nd)
4. Try to get into the stream from any direction
5. Get pushed out of the stream

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.4.0 (unstable)

### -

v1.6.16.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Should be fixed now in the unstable build we just released.

**Reply by FlashORC** [reply]
Thanks for the fix! Works perfectly.
